debian/changelog

```diff
@@ -1,3 +1,49 @@
+libsilkit (4.0.42-1) unstable; urgency=medium
```

Comment:
IIRC, we could just add an 'initial package' message here and drop the duplicates from our CHANGELOG.rst?

Reply:
Absolutely correct